lib/iris/analysis/_interpolate_private.py

```diff
@@ -465,7 +465,7 @@ def regrid(source_cube, grid_cube, mode='bilinear', **kwargs):
 
     """
     if mode == 'bilinear':
-        scheme = iris.analysis.Linear(**kwargs)
+        scheme = Linear(**kwargs)
```

Comment:
NOTE: 
existing version is "technically" wrong, because iris.analysis is not itself imported.
Not a real practical problem, though, as iris itself was imported.